java-strings/src/test/java/com/baeldung/string/ReplaceCharInStringTest.java

```diff
@@ -0,0 +1,23 @@
+package com.baeldung.string;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class ReplaceCharInStringTest {
+    private ReplaceCharacterInString characterInString = new ReplaceCharacterInString();
+
+    @Test
+    public void replaceCarAtIndex(){
```

Comment:
Please review the guidelines for test method naming conventions. The pattern is givenX_whenY_thenZ. The "givenX" part is optional.